testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java

```diff
@@ -760,4 +762,22 @@ public void testDeleteOldCompletedSubscription() {
         subscriber = subscriberDataService.findByCallingNumber(2000000000L);
         assertEquals(1, subscriber.getSubscriptions().size());
     }
+    
+    @Ignore
+    @Test
+    public void verifyFT153() {
+    	/*
```

Comment:
@srivastavaAbhi This is more to do with conventions and keeping up with coding style of the rest of our code base. Method and test descriptions go on top of the method with internal comments being specific to a particular line(s).